static/app/utils/discover/discoverQuery.tsx

```diff
@@ -81,7 +81,10 @@ export function useDiscoverQuery(props: Omit<DiscoverQueryComponentProps, 'child
     };
   };
 
-  return useGenericDiscoverQuery<TableData, DiscoverQueryPropsWithThresholds>({
+  return useGenericDiscoverQuery<
+    TableData & {pageLinks?: string},
```

Comment:
Not sure if I should just combine this with `TableData` rn, as its used in a couple places